src/samVisualize/samVisualizeTypes.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * Enumerates possible types of connections between resources.
+ */
+export enum LinkTypes {
```

Comment:
This is only being used in cfnTemplateGraphGenerator.ts. Just declare the enum there and remove this file.

Reply:
It's used in the rendering code as well, but I have yet to create a PR for that